src/profiler.cpp

```diff
@@ -1077,6 +1077,11 @@ Error Profiler::start(Arguments& args, bool reset) {
         return Error("Profiler already started");
     }
 
+    // Try loading JVM if not already loaded
```

Comment:
The comment is misleading: `tryAttach` does not load a JVM, it rather checks if the JVM is already loaded.